src/Features/CSharp/Portable/Completion/CompletionProviders/OperatorsAndIndexer/UnnamedSymbolCompletionProvider_Operators.cs

```diff
@@ -104,6 +104,11 @@ private void AddOperatorGroup(CompletionContext context, string opName, IEnumera
 
             var displayText = GetOperatorText(opName);
 
+            using var _ = ArrayBuilder<KeyValuePair<string, string>>.GetInstance(out var builder);
```

Comment:
nit: try to give an initial count here.  in future we'll ideally just use a collection expression a swell.